### DISCUSSION TITLE
Display inline calendar as a "grid"

### DISCUSSION THREAD

**Discussion by ibadass** [primefaces/community#1508] [PrimeNG] [answered]
Hello,

I have a calendar with inline option true, numberOfMonths to 14.
Is there a way to display this calendar as a grid of 3 months in each row ?

Thanks.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's not available, however, it might be done by custom css.

**Reply by ibadass** [reply]
Hi,

Thanks for your reply !
Is it possible to help me to do that ?
